src/Examples/Flowsheets/ngcc/hrsg.py

```diff
@@ -1260,9 +1273,9 @@ def _set_fg_port(port, temperature, pressure, flow=fg_rate, fix=False):
         self.split_fg_lp.inlet.temperature[0].unfix()
 
         gas_streams = [
-            #self.g09,
-            #self.g10,
-            #self.g11,
+            # self.g09,
```

Comment:
Is there a reason why these lines are commented out? If there is a reason to keep them, a comment on why would be helpful.